Nodejs/Product/Nodejs/Project/NodejsProjectNode.cs

```diff
@@ -694,7 +657,7 @@ public NodejsProjectNode(NodejsProjectPackage package)
             if (null == ModulesNode) {
                 ModulesNode = new NodeModulesNode(this);
                 AddChild(ModulesNode);
-                CreateIdleNodeModulesWatcher();
+                _idleNodeModulesTimer = new Timer(OnIdleNodeModules);

```

Comment:
I wouldn't create a new timer each time, but reset/start the existing timer... they are not cheap.